src/cpr_sdk/models/search.py

```diff
@@ -547,6 +609,22 @@ def from_vespa_response(cls, response_hit: dict) -> "Document":
 class Passage(Hit):
     """A passage search result hit."""
 
+    class ConceptVersion(BaseModel):
+        concept_id: str
+        canonical_id: str
+
+    class Span(BaseModel):
+        start: int
+        end: int
+
+    class ConceptInstance(BaseModel):
+        concept_id: str
+        model_id_all: str
+        spans_by_model_version: dict[str, list["Span"]]
+
+    concepts_instances: dict[str, ConceptInstance] | None = None
+    concepts_versions: dict[str, str] | None = None
```

Comment:
Pydantic doesnt like this for me and errors on validation, is it meant to be:
```suggestion
    concepts_versions: dict[str, dict[str, str]] | None = None
```

Reply:
Same on the Document object and ModelProfile